### PR TITLE
add Y210/AYUV/Y410 format support in vaDeriveImage.

### PR DESCRIPTION
**Pull request by feiwan1**
Signed-off-by: Wangfei <fei.w.wang@intel.com>

### CODE REVIEW

File: media_driver/linux/common/ddi/media_libva.cpp [hunk 1]
```diff
@@ -4220,6 +4220,9 @@ VAStatus DdiMedia_DeriveImage (
         break;
     case Media_Format_X8R8G8B8:
     case Media_Format_X8B8G8R8:
+    case Media_Format_Y210:
```
Comment (FurongZhang) [inline media_driver/linux/common/ddi/media_libva.cpp]: According to my understanding, Y210 is not 32 bits per pixel.

Comment (feiwan1) [reply media_driver/linux/common/ddi/media_libva.cpp]: @FurongZhang , In my understanding, Y210 is ordered as YUYV..., each YUYV stands for 2 pixel, and each channel has 16bit. so the per_pixel = 4*16/2 =32.

**Comment by wangyan-intel** [comment]
@feiwan1 Is it duplicated with https://github.com/intel/media-driver/pull/555? If yes, please close it.

**Comment by feiwan1** [comment]
Thanks @wangyan-intel for your information. #555 only covered AYUV format in DeriveImage. So I will modify my patch to support Y210&Y410.

**Comment by wangyan-intel** [comment]
#601 #602 #603 are closed. So I add verifying flag for starting merging process. Thanks.

**Pull request merged**